### ISSUE TITLE
Grunt (with grunt-contrib-watch) is using 2.5% of my CPU, would increasing the options.interval value reduce the CPU usage?

### ISSUE THREAD

**Issue by acrolink** [gruntjs/grunt-contrib-watch#504]
I am considering increasing the options.interval to 2000 ms. This means, if I understand correctly, that the watch worker will check the files for changes every 2 seconds. I assume this should reduce the CPU usage, is that correct? Thanks.

**Comment by callmevlad** [comment]
Yes, see this related issue: https://github.com/gruntjs/grunt-contrib-watch/issues/429
